common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/OneForOneBlockFetcher.java

```diff
@@ -100,7 +114,14 @@ public void onSuccess(ByteBuffer response) {
           // Immediately request all chunks -- we expect that the total size of the request is
           // reasonable due to higher level chunking in [[ShuffleBlockFetcherIterator]].
           for (int i = 0; i < streamHandle.numChunks; i++) {
-            client.fetchChunk(streamHandle.streamId, i, chunkCallback);
+            if (fetchToDisk) {
+              final File targetFile = new File(".",
```

Comment:
@cloud-fan 
Yes, but `OneForOneBlockFetcher` is in `network-shuffle` package, I find it hard to import `SparkEnv` from `core` package. Did I miss something?(I'm sorry if this question is stupid.)